php/commands/user.php

```diff
@@ -1120,6 +1120,120 @@ class User_Term_Command extends \WP_CLI\CommandWithTerms {
 	protected $obj_type = 'user';
 }
 
+/**
+ * Manage a user's sessions.
+ *
+ * ## EXAMPLES
+ *
+ */
+class User_Session_Command extends WP_CLI_Command {
+
+	private $fields = array(
+		'token',
+		'login_time',
+		'expiration_time',
+		'ip',
+		'ua',
+	);
+
+	public function __construct() {
+		$this->fetcher = new \WP_CLI\Fetchers\User;
+	}
+
+	/**
+	 * Destroy all sessions for the given user.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <user>
+	 * : User ID, user email, or user login.
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     # Destroy a user's sessions
+	 *     $ wp user session destroy-all admin
+	 *     Success: Destroyed all sessions.
+	 *
+	 * @subcommand destroy-all
```

Comment:
Rather than having a dedicated subcommand for this, could we instead make this `destroy --all`? See #3116